.travis.yml

```diff
@@ -21,6 +22,8 @@ matrix:
       env: SYMFONY_VERSION="2.8.*@dev"
     - php: 5.6
       env: SYMFONY_VERSION="3.0.*@dev"
```

Comment:
please also remove this one. Testing against an EOLed version does not make sense.

My rule is to list only LTS versions explicitly here (the latest version will be tested by the normal jobs)